app/models/admin_auction_status_presenter_factory.rb

```diff
@@ -77,6 +77,18 @@ def available?
     bidding_status.available?
   end
 
+  def over?
+    bidding_status.over?
+  end
+
+  def bids?
+    auction.bids.any?
+  end
+
+  def won?
+    over? && bids?
```

Comment:
I generally order methods above the methods that they are dependent on....so would put these two methods below `won?`